src/main/java/net/dv8tion/jda/core/entities/Invite.java

```diff
@@ -0,0 +1,275 @@
+     *         if the account does not have {@link net.dv8tion.jda.core.Permission#MANAGE_SERVER MANAGE_SERVER} in the invite's channel
+     *
+     * @return {@link net.dv8tion.jda.core.requests.RestAction RestAction} -
+     *
+     *         <pre>
+     * Type: {@link net.dv8tion.jda.core.entities.Invite Invite}
+     *         <br>Value: {@code The deleted Invite object}
+     *         </pre>
+     */
+    public RestAction<Invite> delete();
+
+    /**
+     * Tries to return a new expanded {@link Invite} with more info.
+     * <br>Requires either {@link net.dv8tion.jda.core.Permission#MANAGE_SERVER MANAGE_SERVER} in the invite's guild or
+     * {@link net.dv8tion.jda.core.Permission#MANAGE_CHANNEL MANAGE_CHANNEL} in the invite's channel.
+     * Will throw a {@link net.dv8tion.jda.core.exceptions.PermissionException PermissionException} otherwise.
+     *
+     * @throws net.dv8tion.jda.core.exceptions.PermissionException
+     *         if the account neither has {@link net.dv8tion.jda.core.Permission#MANAGE_SERVER MANAGE_SERVER} in the invite's guild nor
+     *         {@link net.dv8tion.jda.core.Permission#MANAGE_CHANNEL MANAGE_CHANNEL} in the invite's channel
+     *
+     * @return {@link net.dv8tion.jda.core.requests.RestAction RestAction} -
+     *
+     *         <pre>
+     * Type: {@link net.dv8tion.jda.core.entities.Invite Invite}
+     *         <br>Value: {@code The expanded Invite object}
+     *         </pre>
+     *
+     * @see Invite#isExpanded()
+     */
+    public RestAction<Invite> expand();
+
+    /**
+     * Returns an {@link Invite.Channel} object containing info about the invite's channel.
+     *
+     * @return The info about the invite's channel
+     */
+    Channel getChannel();
+
+    /**
+     * Returns the invite code
+     *
+     * @return the invite code
+     */
+    String getCode();
+
+    /**
+        * Returns creation date of this invite.
+        * <br>
+        * <br>This works only for expanded invites and will throw a {@link IllegalStateException} otherwise!
+        *
+        * @throws IllegalStateException if this invite is not expanded
+        *
+        * @return The creation date of this invite
+        *
+        * @see Invite#expand()
+        * @see Invite#isExpanded()
+        */
+    OffsetDateTime getCreationTime();
+
+    /**
+     * Returns an {@link Invite.Guild} object containing info about the invite's guild.
+     *
+     * @return The info about the invite's guild
+     */
+    Guild getGuild();
+
+    /**
+     * Returns the user who created this invite. This may be a fake user.
+     * <br>
+     * <br>This works only for expanded invites and will throw a {@link IllegalStateException} otherwise!
+     *
+     * @throws IllegalStateException if this invite is not expanded
+     *
+     * @return The user who created this invite
+     *
+     * @see Invite#expand()
+     * @see Invite#isExpanded()
+     */
+    User getInviter();
```

Comment:
`getInviter` or `getCreator()` is the better naming here as to not possibly clash with `Owner#getOwner()`